### DISCUSSION TITLE
Error while building bladebit_cuda on ubuntu

### DISCUSSION THREAD

**Discussion by Majesty2699** [Chia-Network/bladebit#449] [General]
Hello everyone, 
I have encountered an error while building bladebit_cuda on Linux/Ubuntu. Building bladebit works without any issues, but when building bladebit_cuda, I receive the message "*** No rule to make target 'bladebit_cuda'. Stop."
Also, retrieving "bladebit help cudaplot" does not work either.

**Comment by haorldbchi** [comment]
This is likely because the cuda toolkit is not installed or it could not be found by cmake, so the target `bladebit_cuda` was not build. Please check the build configuration log to see.

**Comment by Majesty2699** [comment]
Yes, I just took another look at it. During the installation of the toolkit, there is a message stating that not all packages can be installed because they are defective. However, I got these packages from the Nvidia website, and they are described there exactly as required. I'll look up the message again in a moment. Thanks.

**Comment by Majesty2699** [comment]
sudo apt-get -y install cuda-toolkit-12-3
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Some packages could not be installed. This may mean that
you have requested an impossible situation or if you are
using the Unstable distribution that some required packages
have not yet been created or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 nsight-systems-2023.3.3 : Depends on: libtinfo5 but it is not installable
E: Unable to correct problems, you have held broken packages.

**Comment by Majesty2699** [comment]
The toolkit is installed, but bladebit_cuda is still not found. I executed the shell script, and no errors were displayed during the process.

All folders for CUDA were created, and the console also indicated that these were linked.
BladeBit_CUDA is not being built.
